frontend/src/App.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { Routes } from 'react-router-dom';
+import { ToastContainer } from 'react-toastify';
 import './App.css';
-
+import useActionItems from "./hooks/useActionItems";
+import {Route} from "react-router";
+import ActionItemsGallery from "./components/ActionItemsGallery";
+import 'react-toastify/dist/ReactToastify.css';
 function App() {
+    const {actionItems} = useActionItems()
   return (
     <div className="App">
-     Alles raus was keine Miete zahlt!
+        <ToastContainer/>
```

Comment:
Den Toastcontainer könnte man ans Ende der App-div stellen, da dieser ja nicht aktiv als Komponente angezeigt wird und an dieser Stelle möglicherweise etwas in die Übersichtlichkeit grätschen könnte - ist aber optional